import pytest

def test_case():
    with pytest.raises(Exception):
        raise Exception
